RELEASE_NOTES.md

```diff
@@ -6,18 +6,27 @@ Any new bugs, requests, or missing features should be reported as [Github issues
 
 ## Improvements:
 
-### Command line
- * Rename script to zcbor.
+### CDDL language
+ * Add support for 32-bit and 64-bit floating point numbers.
+ * Add support for `undefined` in CDDL.
+ * Add the CDDL prelude (from RFC 8610 Appendix D) to all builds (unless excluded with `--no-prelude`)
+
+
+### Command line and Python module
+ * Rename script/module to zcbor.
  * Better 64 bit support:
    * New argument: `--default-bit-size` which controls the assumed size of integers. See zcbor code `--help` for more information.
+ * Add the --include-prefix option for path prefix to generated h files.
```

Comment:
nit: Should we have this `--include-prefix` in inline syntax?

Reply:
done